### DISCUSSION TITLE
Beta - Editing modes names confusing

### DISCUSSION THREAD

**Discussion by slungs** [home-assistant/frontend#23032] [Automations / scripts / scenes]
A really nice change, but, for me the naming of the two modes is a bit confusing. Wouldn't it be much more logical if "review mode" is the mode where the lights are actually changed so you can review your changes, and edit mode where you... edit? As in a edit > review > apply funnel.

**Comment by marcinbauer-ohf** [comment]
@slungs thanks for commenting. The main cause of changes for the scenes editor is to have the ability to edit them w/o applying (launching) the scene. The added step of "Review mode" aims to achieve this.

**Comment by slungs** [comment]
@marcinbauer85 Yes I understand. My point it that they should be named the other way around. Edit mode = lights not affected. Review mode = scene active while editing, so that you can _review your changes_.
